### DISCUSSION TITLE
Vector Field really required for a collection ?

### DISCUSSION THREAD

**Discussion by dexarry** [milvus-io/milvus#34927] [Q&A and General discussion]
Hello, I was wondering if it's possible to create a collection without a vector field. If so, I haven't found out how. My goal is to create two collections: one with smaller chunks that will be used for vector search and associated with larger chunks in another collection, which will only be retrieved for additional context for the LLM. Therefore, I don't need vectors for that collection. Thank you very much.

**Comment by yhmo** [comment]
Basically, milvus is an in-memory vector database, the major function is ANN search for vectors. So, each collection must require at least a vector field. 
You can store the text content in MySQL. If you don't like to maintain two databases, you can create a collection with 2-dimension vector field to store the text content. The 2-dimension vector field is just a placeholder and does nothing.

**Comment by xiaofan-luan** [comment]
1. you can store chunk in each entity and query by doc to get all vectors.
2. you can maintain the docID and retrieve from object storage
